ci/dev/test-integration.sh

```diff
@@ -24,6 +24,13 @@ main() {
     path="$CODE_SERVER_PATH"
   fi
 
+  if [[ ! -f "test/integration/test-plugin/out/index.js" ]]; then
+    echo "Building test plugin"
+    pushd test/integration/test-plugin
+    make -s out/index.js
```

Comment:
Make seems to be failing in CI, might need to install buildtools?

Reply:
I saw that! Super weird because I see this: 

```
          yum install -y devtoolset-9-{make,gcc,gcc-c++} jq rsync python3
```

Online though it says to do: 
```
yum groupinstall "Development Tools"
```

If it happens again, I'll try that.